src/main/java/com/dinegood/springbootproject/SpringbootprojectApplication.java

```diff
@@ -0,0 +1,63 @@
+package com.dinegood.springbootproject;
```

Comment:
This file should not have any kind of business logic, it should just run the application. Remove everything else thats not needed